tests/DocumentationExamplesTest.php

```diff
@@ -1180,6 +1184,223 @@ public function testIndex_example_2()
         $this->assertEquals('cuisine_1_name_1', $indexName);
     }
 
+    // Start Transactions Intro Example 1
+    private function updateEmployeeInfo1(Manager $manager, Session $session)
+    {
+        $employees = (new Database($manager, "hr"))->employees;
+        $events = (new Database($manager, "reporting"))->events;
+
+        $session->startTransaction([
+            'readConcern' => new ReadConcern("snapshot"),
+            'writeConcern' => new WriteConcern(WriteConcern::MAJORITY)
+        ]);
+
+        try {
+            $employees->updateOne(
+                ['employee' => 3],
+                ['$set' => ['status' => 'Inactive']],
+                ['session' => $session]
+            );
+            $events->insertOne(
+                ['employee' => 3, 'status' => [ 'new' => 'Inactive', 'old' => 'Active']],
+                ['session' => $session]
+            );
+        } catch (\MongoDB\Driver\Exception\Exception $error) {
+            echo "Caught exception during transaction, aborting.\n";
+            $session->abortTransaction();
+            throw $error;
+        }
+
+        while (true) {
+            try {
+                $session->commitTransaction();
+                echo "Transaction committed.\n";
+                break;
+            } catch (\MongoDB\Driver\Exception\CommandException $error) {
+                $resultDoc = $error->getResultDocument();
+
+                if (isset($resultDoc->errorLabels) && in_array('UnknownTransactionCommitResult', $resultDoc->errorLabels)) {
+                    echo "UnknownTransactionCommitResult, retrying commit operation ...\n";
+                    continue;
+                } else {
+                    echo "Error during commit ...\n";
+                    throw $error;
+                }
+            } catch (\MongoDB\Driver\Exception\Exception $error) {
+                echo "Error during commit ...\n";
+                throw $error;
+            }
+        }
+    }
+    // End Transactions Intro Example 1
+
+    public function testTransactions_intro_example_1()
+    {
+        $hrDb = new Database($this->manager, "hr");
+        $reportingDb = new Database($this->manager, "reporting");
+
+        $hrDb->dropCollection('employees');
+        $reportingDb->dropCollection('events');
+
+        /* Collections need to be created before a transaction starts */
+        $hrDb->createCollection('employees');
+        $reportingDb->createCollection('events');
+
+        $session = $this->manager->startSession();
```

Comment:
Be sure to use the Client object here. Although the Client and Manager share the same sockets (same URI), they are different Manager instances (Client creates its own internally) and you'd get a driver exception if a Session from one Manager is used with another.

Reply:
Done